grpc-google-cloud-webrisk-v1beta1/src/main/java/com/google/webrisk/v1beta1/WebRiskServiceV1Beta1Grpc.java

```diff
@@ -161,30 +137,20 @@ private WebRiskServiceV1Beta1Grpc() {}
     return getSearchUrisMethod;
   }
 
-  @io.grpc.ExperimentalApi("https://github.com/grpc/grpc-java/issues/1901")
-  @java.lang.Deprecated // Use {@link #getSearchHashesMethod()} instead.
-  public static final io.grpc.MethodDescriptor<
-          com.google.webrisk.v1beta1.SearchHashesRequest,
-          com.google.webrisk.v1beta1.SearchHashesResponse>
-      METHOD_SEARCH_HASHES = getSearchHashesMethodHelper();
-
   private static volatile io.grpc.MethodDescriptor<
           com.google.webrisk.v1beta1.SearchHashesRequest,
           com.google.webrisk.v1beta1.SearchHashesResponse>
       getSearchHashesMethod;
 
-  @io.grpc.ExperimentalApi("https://github.com/grpc/grpc-java/issues/1901")
+  @io.grpc.stub.annotations.RpcMethod(
```

Comment:
just want to confirm these are intended changes as part of adding a v1 client?

Reply:
This is an artifact of switching to the bazel generator. I can separate this into a different PR

Reply:
Your call, just wanted to confirm! I'll approve this PR, feel free to separate out if needed

Reply:
Removed the v1beta1 stuff and will do it separately for a cleaner commit history